src/Attendance/AttendanceOtherForm/AttendanceOtherForm.js

```diff
@@ -211,45 +214,58 @@ export function AttendanceOtherForm({
               inline
             />
             {values.pay && (
-              <FieldWithError
-                errors={errors}
-                name="absentReason"
-                component={RadioGroup}
-                label="Select a reason"
-                options={getAbsentReasons(values.pay).map((r) => ({ title: r.name, value: r.value }))}
-              />
-            )}
-            {values.pay && shouldTriggerSubReason(values.absentReason) && (
-              <ConditionalRadio>
-                <FieldWithError
-                  errors={errors}
-                  name="absentSubReason"
-                  component={Select}
-                  label="Select an absence reason"
-                >
-                  <option value="" disabled>
-                    Select
-                  </option>
-                  {getAbsentSubReasons(values.pay, values.absentReason).map((reason) => (
-                    <option key={reason.value} value={reason.value}>
-                      {reason.name}
-                    </option>
-                  ))}
-                </FieldWithError>
-                {shouldTriggerIEP(values.absentReason) && (
-                  <FieldWithError
-                    name="iep"
-                    errors={errors}
-                    component={RadioGroup}
-                    label="Do you want to add an incentive level warning"
-                    options={[
-                      { title: 'Yes', value: 'yes' },
-                      { title: 'No', value: 'no' },
-                    ]}
-                    inline
-                  />
-                )}
-              </ConditionalRadio>
+              <FieldWithError errors={errors} name="absentReason" component={MultiChoice} label="Select a reason">
```

Comment:
had to split out the radiogroup into constitutent parts i.e. multichoice with radios inside